packages/astro/performance/content-benchmark.mjs

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable no-console */
```

Comment:
Maybe we can migrate this performance directory to the new `/benchmark` directory at the later stage (happy to help too), but I think it's great to check this performance in at the meantime.